hail/src/test/scala/is/hail/TestUtils.scala

```diff
@@ -414,7 +415,14 @@ object TestUtils {
     val t = x.typ
     assert(t.typeCheck(expected), t)
 
-    ExecStrategy.values.intersect(execStrats).foreach { strat =>
+    val filteredExecStrats: Set[ExecStrategy] =
+      if (HailContext.backend.isInstanceOf[SparkBackend]) execStrats
+      else {
+        info("skipping interpret and non-lowering compile steps on non-spark backend")
+        execStrats.intersect(ExecStrategy.backendOnly)
+      }
```

Comment:
You didn't like my idea of a BackEnd.supportedStrats?

Reply:
Hmm, I guess I saw ExecStrategy as more of a testing utility (since that's the only place it's used) and was reluctant to put test-specific stuff on the backend. That doesn't necessarily have to be the case, though---we could start using it to control (from python?) what strategy we're using to execute a pipeline. I think the set of strategies we'd want for that are perhaps slightly different than the ones we want for testing right now.

Reply:
Yeah, good point.  ExecStrategy, in particular, is defined in the test module, so it shouldn't go on Backend.  I'm happy with this change now, thanks for clarifying.